### PR TITLE
update nightly tests jobs

### PR DESCRIPTION
**Pull request by meldafrawi**

### CODE REVIEW

**Comment by meldafrawi** [comment]
will fix https://ci.longhorn.io/job/public/job/master/job/ubuntu/job/amd64/job/longhorn-upgrade-tests-ubuntu-amd64/504/

Ref: https://ci.longhorn.io/job/private/job/longhorn-tests-regression/530/

File: jenkins-jobs/master/ubuntu/longhorn-tests-ubuntu-amd64.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 - job:
-    name: longhorn-tests-ubuntu-amd64
+    name: longhorn-tests-rockylinux-amd64
```
Comment (khushboo-rancher) [inline jenkins-jobs/master/ubuntu/longhorn-tests-ubuntu-amd64.yml:2]: Isn't this ubuntu file, why have we changed the name?

File: jenkins-jobs/master/ubuntu/longhorn-tests-ubuntu-amd64.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
     project-type: pipeline
-    folder: public/master/ubuntu/amd64
+    folder: public/master/rockylinux/amd64
```
Comment (khushboo-rancher) [inline jenkins-jobs/master/ubuntu/longhorn-tests-ubuntu-amd64.yml:4]: Same as above comment

**Review by khushboo-rancher — CHANGES_REQUESTED** [review]
I think the ubuntu-amd64.yml has the rockyLinux changes by mistakes

**Comment by meldafrawi** [comment]
Sorry, copy and paste typo
Fixed, Thanks @khushboo-rancher

File: jenkins-jobs/master/ubuntu/longhorn-tests-ubuntu-amd64.yml [hunk 3]
```diff
@@ -106,9 +109,3 @@
               - master
       script-path: test_framework/Jenkinsfile
       lightweight-checkout: true
-    triggers:
```
Comment (khushboo-rancher) [inline jenkins-jobs/master/ubuntu/longhorn-tests-ubuntu-amd64.yml:109]: Just to confirm, are we removing the auto trigger part from ubuntu amd job?

**Pull request merged**